analysis/process.cc

```diff
@@ -436,6 +436,10 @@ int main(int argc, char** argv)
 
 			   return passTrig;
 		       }, UNITY);
+    ana.cutflow.addCutToLastActiveCut("CutNoiseFilter", [&]()
+                                      {
+                                          return vvv.Common_noiseFlag();
```

Comment:
this is the noise flag that is applied for data. now we apply for both data and MC.